src/validation.h

```diff
@@ -826,7 +826,7 @@ class ChainstateManager
     std::thread m_load_block;
     //! A single BlockManager instance is shared across each constructed
     //! chainstate to avoid duplicating block metadata.
-    BlockManager m_blockman GUARDED_BY(::cs_main);
+    BlockManager m_blockman{};
```

Comment:
Why add the braces? `m_blockman` is initialized in the ctor.

Reply:
This *is* the ctor

Reply:
I liked them, because they clarified that there is no ctor and the initialization happens inline

Reply:
> I liked them, because they clarified that there is no ctor and the initialization happens inline

I wouldn't object to adding them back if people have strong opinions, but they don't guarantee this. Someone could add a ChainStateMananger constructor in the future which does pass arguments to the m_blockman constructor, and this code would still compile as long as m_blockman also has a default constructor. The nice thing about plain `BlockManager m_blockman;` is it says "ChainStateMananger has a m_blockman member. If you care about how it will be constructed, go look at the constructor implementation, like you need to do anyway."

Reply:
Oh indeed, thanks for the responses!